changelog.d/pa-2208.fixed

```diff
@@ -0,0 +1 @@
+Taint: Fixed an issue where findings with the same sink would only report one finding
```

Comment:
It could be explained a bit better using some more words? I understand what it means but I'm not sure @adamkvitek will understand it.